skopt/acquisition.py

```diff
@@ -34,23 +34,40 @@ def _gaussian_acquisition(X, model, y_opt=None, acq_func="LCB",
     kappa = acq_func_kwargs.get("kappa", 1.96)
 
     # Evaluate acquisition function
-    if acq_func == "LCB":
-        return gaussian_lcb(X, model, kappa, return_grad)
+    per_second = acq_func.endswith("ps")
+    if per_second:
+        model, time_model = model.estimators_
 
-    elif acq_func in ["EI", "PI"]:
-        if acq_func == "EI":
+    if acq_func in ["LCB", "LCBps"]:
+        func_and_grad = gaussian_lcb(X, model, kappa, return_grad)
+        if return_grad:
+            acq_vals, acq_grad = func_and_grad
+        else:
+            acq_vals = func_and_grad
+
+    elif acq_func in ["EI", "PI", "EIps", "PIps"]:
+        if acq_func in ["EI", "EIps"]:
             func_and_grad = gaussian_ei(X, model, y_opt, xi, return_grad)
         else:
             func_and_grad = gaussian_pi(X, model, y_opt, xi, return_grad)
 
         if return_grad:
-            return -func_and_grad[0], -func_and_grad[1]
+            acq_vals = -func_and_grad[0]
+            acq_grad = -func_and_grad[1]
         else:
-            return -func_and_grad
+            acq_vals = -func_and_grad
 
     else:
         raise ValueError("Acquisition function not implemented.")
 
+    if return_grad:
+        return acq_vals, acq_grad
+
+    if per_second:
+        mean, std = time_model.predict(X, return_std=True)
+        inv_t = np.exp(-mean + 0.5*std**2)
+        return acq_vals * inv_t
```

Comment:
Clarification, "Maybe an exception would be nice here": in case return_grad = True, but ps acquisition is used